### DISCUSSION TITLE
GitHub Explore, Filter by star count

### DISCUSSION THREAD

**Discussion by JS555** [community/community#21727] [New to GitHub]
<p>When initially in the “Explore” pane of GitHub. Is there a way to filter the listings by star count. There is currently no filter it seems. I have seen mentions of filtering searches by star count. But is there no way to filter “Explore” which perhaps shows every project existing on Github, by star count. </p>
<p>Also does “Explore” list all projects which have existed, or randomized projects which are currently worked. It would be interesting and perhaps quite useful to see existing projects with the most star count, including projects years old. </p>

**Comment by fire-eggs** [comment, marked as answer]
<ol>
<li>
<p>This forum is for people’s questions regarding their Github based web pages. You probably want the <a href="/t5/How-to-use-Git-and-GitHub/bd-p/Board_1">“how to use github”</a> forum.</p>
</li>
<li>
<p>“Trending” on Explore is showing most-starred repos for the last day/week/month.</p>
</li>
<li>
<p>See <a href="https://gitstar-ranking.com/" rel="nofollow noopener">this page</a> or use <a href="https://github.com/search?q=stars%3A%3E0&amp;s=stars&amp;type=Repositories" rel="nofollow noopener">this</a> for all-time star rankings.</p>
</li>
</ol>

**Comment by JS555** [comment]
<p>Ah for sure, thank you. Indeed.</p>

**Comment by AndreaGriffiths11** [comment]
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by JS555** [comment]
<p>Appreciate that</p>
<p>Regards,</p>
<p>John S</p>
